templates/admin/judge/problem/change_form.html

```diff
@@ -0,0 +1,20 @@
+{% extends "admin/change_form.html" %}
+{% load i18n %}
+
+{% block extrahead %}{{ block.super }}
+    <script>
+        django.jQuery(function ($) {
+            $('.submissionslink').appendTo('div#bottombar').show();
+        });
+    </script>
+{% endblock extrahead %}
+
+{% block after_field_sets %}{{ block.super }}
+    {% if original %}
+        <a style="display: none" title="{% trans "View Submissions" %}" href="{% url 'admin:judge_submission_changelist' %}?problem__code={{ original.code }}"
```

Comment:
Line too long, swap `href` with `class` so that `href` gets the whole line to itself.